app/models/spree/line_item_decorator.rb

```diff
@@ -0,0 +1,6 @@
+Spree::LineItem.class_eval do
```

Comment:
I'm cool with consistency.

`order_instance.line_items` returns an `Spree::LineItem` ActiveRelation object, which you can chain additional scopes on. So, if you defined a `with_tax_rates` scope, you could use it with `order_instance.line_items.with_tax_rates`. Alternatively, you could keep your `Order#line_items_with_tax_rates` method, but reduce its implementation to `line_items.with_tax_rates`.